.github/scripts/compare_build_time.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
this file is more like iai-callgrind benchmarks and not compare_build_time as it runs benchmarks for the code right?

for comparing build time we need to measure how long is the compilation of the project

Reply:
made the changes accordingly